build/build-image/cross/Dockerfile

```diff
@@ -43,14 +43,15 @@ RUN apt-get update \
 
 # Use dynamic cgo linking for architectures other than amd64 for the server platforms
 # To install crossbuild essential for other architectures add the following repository.
-RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
+RUN echo "deb [arch=amd64] http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
   && apt-key adv --no-tty --keyserver keyserver.ubuntu.com --recv-keys 40976EAF437D05B5 3B4FE6ACC0B21F32 \
   && apt-get update \
   && apt-get install -y build-essential \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do apt-get install -y crossbuild-essential-${platform}; done \
+  && ([ "$(uname -m)" = "x86_84" ] && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do apt-get install -y crossbuild-essential-${platform}; done || true) \
   && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-RUN PROTOBUF_VERSION=3.0.2; ZIPNAME="protoc-${PROTOBUF_VERSION}-linux-x86_64.zip"; \
+RUN PROTOBUF_VERSION=3.7.0; [ "$(uname -m)" = "x86_84" ] && ZIPNAME="protoc-${PROTOBUF_VERSION}-linux-x86_64.zip" || true; \
+  [ "$(uname -m)" = "aarch64" ] && ZIPNAME="protoc-${PROTOBUF_VERSION}-linux-aarch_64.zip" || true; \
```

Comment:
Please extend the support for `ppc64le` platform as well, here is the file: https://github.com/protocolbuffers/protobuf/releases/download/v3.7.0/protoc-3.7.0-linux-ppcle_64.zip